File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    answer = ''
    new_food = []
    for i in food:
        if (i % 2 != 0):
            i -= 1
        new_food.append(i)
    print(new_food)
    for i in range(1, len(new_food)):
        answer += str(i) * (new_food[i] // 2) 
    answer = answer + "0" + answer[::-1]
    return answer

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    var word = "";
    
    food.map((c, index) => {
        word += index.toString().repeat(Math.floor(c/2));
    })
    
    
    word = word + "0" + [...word].reverse().join("");
    
    return word;
    
}